src/Middleware/NonAvailablePlaceHTMLFormatter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\CalendarSummaryV3\Middleware;
+
+use Closure;
+use CultuurNet\CalendarSummaryV3\Translator;
+use CultuurNet\SearchV3\ValueObjects\Offer;
+use CultuurNet\SearchV3\ValueObjects\Place;
+
+class NonAvailablePlaceHTMLFormatter
+{
+    /**
+     * @var Translator
+     */
+    private $trans;
+
+    public function __construct(
+        string $langCode
+    ) {
+        $this->trans = new Translator();
+        $this->trans->setLanguage($langCode);
+    }
+
+    public function format(Offer $offer, Closure $next): string
+    {
+        if ($offer instanceof Place) {
+            if ($offer->getStatus()->getType() === 'Unavailable') {
```

Comment:
Nit: Maybe we should add these status types as constants? Perhaps on the `Status` object in `search-v3`?

Reply:
Good plan!